:tada: This fails as expected when #1738 is applied:

```
com.microsoft.playwright.TimeoutError: 
Error {
  message='Timeout 30000ms exceeded.
  name='TimeoutError
  stack='TimeoutError: Timeout 30000ms exceeded.
    at ProgressController.run (/tmp/playwright-java-7802024893408311694/package/lib/server/progress.js:92:28)
    at FrameDispatcher._runCommand (/tmp/playwright-java-7802024893408311694/package/lib/server/dispatchers/dispatcher.js:91:31)
    at DispatcherConnection.dispatch (/tmp/playwright-java-7802024893408311694/package/lib/server/dispatchers/dispatcher.js:325:39)
}
Call log:
-   - waiting for getByRole(AriaRole.COMBOBOX).nth(1)

	at com.microsoft.playwright.impl.WaitableResult.get(WaitableResult.java:52)
	at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:138)
	at com.microsoft.playwright.impl.Connection.sendMessage(Connection.java:131)
	at com.microsoft.playwright.impl.ChannelOwner.sendMessage(ChannelOwner.java:124)
	at com.microsoft.playwright.impl.FrameImpl.selectOption(FrameImpl.java:691)
	at com.microsoft.playwright.impl.FrameImpl.selectOptionImpl(FrameImpl.java:670)
	at com.microsoft.playwright.impl.FrameImpl.selectOption(FrameImpl.java:120)
	at com.microsoft.playwright.impl.FrameImpl.selectOption(FrameImpl.java:109)
	at com.microsoft.playwright.impl.LocatorImpl.selectOption(LocatorImpl.java:526)
	at com.microsoft.playwright.Locator.selectOption(Locator.java:4589)
	at org.jenkinsci.plugins.workflow.WorkflowEditorTest.xxx(WorkflowEditorTest.java:55)
Caused by: com.microsoft.playwright.TimeoutError: Error {
  message='Timeout 30000ms exceeded.
  name='TimeoutError
  stack='TimeoutError: Timeout 30000ms exceeded.
    at ProgressController.run (/tmp/playwright-java-7802024893408311694/package/lib/server/progress.js:92:28)
    at FrameDispatcher._runCommand (/tmp/playwright-java-7802024893408311694/package/lib/server/dispatchers/dispatcher.js:91:31)
    at DispatcherConnection.dispatch (/tmp/playwright-java-7802024893408311694/package/lib/server/dispatchers/dispatcher.js:325:39)
}
Call log:
-   - waiting for getByRole(AriaRole.COMBOBOX).nth(1)

	at com.microsoft.playwright.impl.Connection.dispatch(Connection.java:257)
	at com.microsoft.playwright.impl.Connection.processOneMessage(Connection.java:214)
	at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:136)
	... 9 more
```

--- a/plugin/src/main/js/samples.js
+++ b/plugin/src/main/js/samples.js
@@ -3,13 +3,13 @@ import $ from 'jquery';
 export function addSamplesWidget(editor, editorId, samplesUrl) {
     var samples = [];
 
-    if ($('#workflow-editor-wrapper .samples').length) {
+    if ($('#workflow-editor-samples').length) {
         // Already there.
         return;
     }
 
     var $aceEditor = $('#' + editorId);
-    var sampleSelect = $('<select></select>');
+    var sampleSelect = $('<select id="workflow-editor-samples"></select>');
 
     sampleSelect.append('<option >try sample Pipeline...</option>');
     fetch(samplesUrl, {
@@ -26,11 +26,8 @@ export function addSamplesWidget(editor, editorId, samplesUrl) {
         }
     });
 
-    var samplesDiv = $('<div class="samples"></div>');
-    samplesDiv.append(sampleSelect);
-
-    samplesDiv.insertBefore($aceEditor);
-    samplesDiv.css({
+    sampleSelect.insertBefore($aceEditor);
+    sampleSelect.css({
         'position': 'absolute',
         'right': '1px',
         'z-index': 100,

--- a/plugin/src/test/java/org/jenkinsci/plugins/workflow/WorkflowEditorTest.java
+++ b/plugin/src/test/java/org/jenkinsci/plugins/workflow/WorkflowEditorTest.java
@@ -39,6 +39,8 @@ import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
 import org.jenkinsci.plugins.workflow.cps.GroovySample;
 import org.jenkinsci.plugins.workflow.job.WorkflowJob;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
 import org.jvnet.hudson.test.JenkinsRule;
 import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
 
@@ -46,6 +48,7 @@ import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
 @WithJenkins
 class WorkflowEditorTest {
 
+    @DisabledOnOs(OS.WINDOWS) // TODO https://github.com/jenkinsci/workflow-cps-plugin/pull/1775#discussion_r3220669617
     @Test
     void smokes(JenkinsRule r, Page page) throws Exception {
         ExtensionList.lookupSingleton(CpsFlowDefinition.DescriptorImpl.class).enableWorkflowEditor = true;

--- a/plugin/src/test/java/org/jenkinsci/plugins/workflow/WorkflowEditorTest.java
+++ b/plugin/src/test/java/org/jenkinsci/plugins/workflow/WorkflowEditorTest.java
@@ -36,6 +36,7 @@ import com.microsoft.playwright.options.AriaRole;
 import hudson.ExtensionList;
 import java.util.List;
 import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.cps.GroovySample;
 import org.jenkinsci.plugins.workflow.job.WorkflowJob;
 import org.junit.jupiter.api.Test;
 import org.jvnet.hudson.test.JenkinsRule;
@@ -46,25 +47,30 @@ import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
 class WorkflowEditorTest {
 
     @Test
-    void xxx(JenkinsRule r, Page page) throws Exception {
-        page.context().grantPermissions(List.of("clipboard-read", "clipboard-write"));
+    void smokes(JenkinsRule r, Page page) throws Exception {
         ExtensionList.lookupSingleton(CpsFlowDefinition.DescriptorImpl.class).enableWorkflowEditor = true;
         var p = r.createProject(WorkflowJob.class, "p");
         p.setDefinition(new CpsFlowDefinition("", true));
-        page.navigate(p.getAbsoluteUrl() + "configure");
-        page.getByRole(AriaRole.COMBOBOX).nth(1).selectOption("scripted");
-        page.locator(".ace_content").click();
-        page.getByRole(AriaRole.TEXTBOX, new Page.GetByRoleOptions().setName("Cursor at row"))
-                .press("ControlOrMeta+a");
-        page.getByRole(AriaRole.TEXTBOX, new Page.GetByRoleOptions().setName("Cursor at row"))
-                .press("ControlOrMeta+x");
 
-        // Assert that clipboard starts with "node {"
-        String clipboard1 = (String) page.evaluate("navigator.clipboard.readText()");
-        assertThat(clipboard1, startsWith("node {"));
+        // Configure project, select sample, focus editor
+        page.context().grantPermissions(List.of("clipboard-read", "clipboard-write"));
+        page.navigate(p.getAbsoluteUrl() + "configure");
+        page.getByRole(AriaRole.COMBOBOX)
+                .nth(1)
+                .selectOption(ExtensionList.lookupSingleton(GroovySample.Scripted.class)
+                        .name());
+        page.locator(".ace_content").click();
+        var textbox = page.getByRole(AriaRole.TEXTBOX, new Page.GetByRoleOptions().setName("Cursor at row"));
+
+        // Cut the sample to clipboard
+        textbox.press("ControlOrMeta+a");
+        textbox.press("ControlOrMeta+x");
+        assertThat(
+                "looks like scripted.groovy",
+                (String) page.evaluate("navigator.clipboard.readText()"),
+                startsWith("node {"));
 
         // Type "node {\n" again (character by character, not .fill())
-        var textbox = page.getByRole(AriaRole.TEXTBOX, new Page.GetByRoleOptions().setName("Cursor at row"));
         textbox.pressSequentially("node {");
         textbox.press("Enter");
 
@@ -75,12 +81,13 @@ class WorkflowEditorTest {
         textbox.press("ControlOrMeta+a");
         textbox.press("ControlOrMeta+c");
 
-        // Assert clipboard contains the formatted code with proper indentation
-        String clipboard2 = (String) page.evaluate("navigator.clipboard.readText()");
-        assertThat(clipboard2, equalTo("""
-                node {
-                    sh 'echo hello'
-                }"""));
+        assertThat(
+                "clipboard contains the formatted code with proper indentation",
+                (String) page.evaluate("navigator.clipboard.readText()"),
+                equalTo("""
+            node {
+                sh 'echo hello'
+            }"""));
     }
 
     public static final class HeadlessOptionsFactory implements OptionsFactory {

--- a/plugin/src/test/java/org/jenkinsci/plugins/workflow/WorkflowEditorTest.java
+++ b/plugin/src/test/java/org/jenkinsci/plugins/workflow/WorkflowEditorTest.java
@@ -58,8 +58,7 @@ class WorkflowEditorTest {
         // Configure project, select sample, focus editor
         page.context().grantPermissions(List.of("clipboard-read", "clipboard-write"));
         page.navigate(p.getAbsoluteUrl() + "configure");
-        page.getByRole(AriaRole.COMBOBOX)
-                .nth(1)
+        page.locator("#workflow-editor-samples")
                 .selectOption(ExtensionList.lookupSingleton(GroovySample.Scripted.class)
                         .name());
         page.locator(".ace_content").click();

